### PR TITLE
CeTune/README: rm unnecessary works and fix some error describle

### PR DESCRIPTION
**Pull request by yonghengdexin735**
Signed-off-by: zhang.zezhu <zhang.zezhu@zte.com.cn>

### CODE REVIEW

**Comment by xuechendi** [comment]
@yonghengdexin735 , thanks for the correction, but it shows conflicts, can you rebase then push the branch again?

**Comment by yonghengdexin735** [comment]
@xuechendi done

File: README.md [hunk 1]
```diff
@@ -7,13 +7,13 @@
 ####Maintainance
 - CeTune is an opensource project, under LGPL V2.1, Drived by INTEL BDT CSE team.
 - Maillist: https://github.com/01org/CeTune
-- Subscribe maillist: https://lists.01.org/mailman/listinfo/cephperformance
+- Subscribe maillist: https://lists.01.org/mailman/listinfo/cephperformanc
```
Comment (xuechendi) [inline README.md]: ?cephperformanc

**Comment by yonghengdexin735** [comment]
@xuechendi  thanks, it's my fault, done

File: README.md [hunk 2]
```diff
@@ -81,29 +81,29 @@ Assume ceph is installed on all nodes, this part is demonstrate the workflow of
...
-  list_client | ${hostname1},${hostname2},... | List client(rbd/cosbench worker) nodes here, split by ','
-  list_mon | ${hostname1},${hostname2},... | List mon nodes here, split by ','
-  ${server_name} | ${osd_device1}:${journal_device1},${osd_device2}:${journal_device2},... | After adding nodes at 'list_server', cetune will add new lines whose key is the server's name;Add osd:journal pair to corresponding node, split by ','
+  head | ${hostname} | Cetune controller node hostname.
+  user | root | Only support root currently.
+  enable_rgw | true / false | Set true, cetune will also deploy radosgw; Set false, only deploy osd and rbd nodes.
+  list_server | ${hostname1},${hostname2},... | List osd nodes here, split by ','.
+  list_client | ${hostname1},${hostname2},... | List client(rbd/cosbench worker) nodes here, split by ','.
+  list_mon | ${hostname1},${hostname2},... | List mon nodes here, split by ','.
+  ${server_name} | ${osd_device1}:${journal_device1},${osd_device2}:${journal_device2},... | After adding nodes at 'list_server', cetune will add new lines whose key is the server's name;Add osd:journal pair to corresponding node, split by ','.
```
Comment (xuechendi) [inline README.md]: we don't need '.' here, it shows inside the third colume of the table, and the sentence is not complete.

**Comment by yonghengdexin735** [comment]
@xuechendi CeTune can test some client for fio and  collect its information? and how? 
thanks!

**Comment by xuechendi** [comment]
@yonghengdexin735 just add a test case under benchmark config as 'fiorbd', and follow the instruction to fill all the highlight rows.

**Comment by yonghengdexin735** [comment]
@xuechendi like: one client and two server
client for CeTune, server  ceph_client for fio test
so how to collect data for two fio?

if I test for fio like:one client and two server. Using: `fio --client=<server0> fio0.conf --client=<server1> fio1.conf --group-repoting` can collect information for fio, but I don't know how to use CeTune to do the same thing

**Comment by xuechendi** [comment]
@yonghengdexin735 , add the nodes who will run fio at list_clients , and the nodes who will act as server at list_server, you can add same hostname in both list_client and list_server.
And when you need to run two fio jobs on two hosts, make workers in testcase config as 2, so cetune will distribute the job to list_client

**Comment by yonghengdexin735** [comment]
@xuechendi thanks for your share, I will do it

**Comment by yonghengdexin735** [comment]
@xuechendi hi, I run two client,but the status for CeTune is drop-dead halt for 1 job, 
like:`[WARNING]: 34 fio job still runing
[WARNING]: 34 fio job still runing
[WARNING]: 1 fio job still runing
[WARNING]: 1 fio job still runing
[WARNING]: 1 fio job still runing
[WARNING]: 1 fio job still runing
[WARNING]: 1 fio job still runing
[WARNING]: 1 fio job still runing`

and the leader client can collect the second client msg for log

**Comment by xuechendi** [comment]
@yonghengdexin735 , that is possibly there is one issue with that job/rbd volume which cause fio unable to close the rbd image. If you just care the performance, you can using cancel button to interrupt this test, and cetune will help to collect the data and do analyzing after all.

**Comment by xuechendi** [comment]
@yonghengdexin735 , you can send me email by chendi.xue@intel.com, this PR is done, I don't think this is a good idea discussing other issues here

**Comment by yonghengdexin735** [comment]
@xuechendi , sorry I have not outgoing Email permission.
 I have two clients but having different kernel. when I run CeTune. One job cann't kill. One of kernel have 'dm_bufio_cache' process, so the code of 'killall -9 fio' cann't kill 'dm_bufio_cache' process (the 'dm_bufio_cache' process is always being ). I picked up your comment to interrupt this test and cetune will help to collect the data and do analyzing after all.

**Pull request merged**